es-bs-base/scss/_forms.scss

```diff
@@ -345,3 +345,9 @@ textarea.form-control {
     }
   }
 }
+
+// NOTE: this loading UI is subject to change
```

Comment:
Subject to change, via https://energysage.atlassian.net/wiki/spaces/DS/pages/198311937/2023-01-31+Loading+UI+User+feedback+for+long-running+requests+Upcoming#%E2%A4%B4-Decisions